modules/unsupported/mbstyle/src/main/java/org/geotools/mbstyle/function/MBFunction_any.java

```diff
@@ -0,0 +1,63 @@
+/*
+ *    GeoTools - The Open Source Java GIS Toolkit
+ *    http://geotools.org
+ * 
+ *    (C) 2018, Open Source Geospatial Foundation (OSGeo)
+ * 
+ *    This library is free software; you can redistribute it and/or
+ *    modify it under the terms of the GNU Lesser General Public
+ *    License as published by the Free Software Foundation;
+ *    version 2.1 of the License.
+ * 
+ *    This library is distributed in the hope that it will be useful,
+ *    but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ *    Lesser General Public License for more details.
+ */
+package org.geotools.mbstyle.function;
+
+import static org.geotools.filter.capability.FunctionNameImpl.parameter;
+
+import org.geotools.filter.FunctionExpressionImpl;
+import org.geotools.filter.capability.FunctionNameImpl;
+import org.opengis.filter.capability.FunctionName;
+import org.opengis.filter.expression.Expression;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * MapBox Expression function that returns {@link java.lang.Boolean#TRUE} if any expression evaluates to
+ * {@link java.lang.Boolean#TRUE}, {@link java.lang.Boolean#FALSE} otherwise. This function is implemented as a short-
+ * circuit in that it will return {@link java.lang.Boolean#TRUE} for the first expression that evaluates to
+ * {@link java.lang.Boolean#TRUE}.
+ */
+class MBFunction_any extends FunctionExpressionImpl {
```

Comment:
As noted above, this class is unnecessary - its functionality can be replicated by a sequence of ORs